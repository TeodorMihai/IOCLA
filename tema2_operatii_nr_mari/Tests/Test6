85FB7813AAAAAAAAAAAAAAAAAA << 15
