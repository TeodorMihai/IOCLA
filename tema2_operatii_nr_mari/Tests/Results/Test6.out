AAAAAAAAAAAAAAAAAA1378FB85
15
000040555555555555555575026FBF10
10BF6F02755555555555555555400000
